virtualizarr/backend.py

```diff
@@ -133,9 +137,38 @@ def open_virtual_dataset(
         raise NotImplementedError()
 
     # if filetype is user defined, convert to FileType
+
     if filetype is not None:
         filetype = FileType(filetype)
 
+    if filetype == FileType.kerchunk_parquet:
+        from fsspec.implementations.reference import LazyReferenceMapper
+
+        from virtualizarr.readers.kerchunk import dataset_from_kerchunk_refs
+
+        fs = _FsspecFSFromFilepath(filepath=filepath, reader_options=reader_options).fs
+        lrm = LazyReferenceMapper(filepath, fs)
+
+        # build reference dict from KV pairs in LazyReferenceMapper
+        array_refs = {k: lrm[k] for k in lrm.keys()}
+
+        # where does version come from? Is it always 1?
+        full_reference = {"version": 1, "refs": array_refs}
+
+        return dataset_from_kerchunk_refs(KerchunkStoreRefs(full_reference))
```

Comment:
Following the above comment - I guess `dataset_from_kerchunk_refs` should be a common utility that every reader that produces kerchunk-formatted references calls internally before returning a virtual dataset.

Reply:
Seems like a good standardization.

Reply:
Opened up an issue to track this: https://github.com/zarr-developers/VirtualiZarr/issues/257